core/templates/pages/skill-editor-page/modal-templates/skill-editor-save-modal.directive.html

```diff
@@ -6,7 +6,7 @@ <h3>
 <div class="modal-body oppia-long-text">
   <p>
     Please enter a brief description of what you have changed:
-    <textarea rows="3" cols="50" ng-model="commitMessage" focus-on="saveChangesModalOpened" class="protractor-test-commit-message-input" autofocus></textarea>
+    <textarea rows="3" cols="50" maxlength="1000" ng-model="commitMessage" focus-on="saveChangesModalOpened" class="protractor-test-commit-message-input" autofocus></textarea>
```

Comment:
Why is this change required?

Reply:
We made this change so that the user cannot input any text with length larger than 1000, which matches the new limit on commit message sizes. This was something that Sean asked me to add to the PR.